CHANGELOG.md

```diff
@@ -33,6 +33,7 @@ N/A
 - [#1444](https://github.com/oauth2-proxy/oauth2-proxy/pull/1444) Update LinkedIn provider validate URL (@jkandasa)
 - [#1471](https://github.com/oauth2-proxy/oauth2-proxy/pull/1471) Update alpine to 3.15 (@AlexanderBabel)
 - [#1479](https://github.com/oauth2-proxy/oauth2-proxy/pull/1479) Update to Go 1.17 (@polarctos)
+- [#1454](https://github.com/oauth2-proxy/oauth2-proxy/pull/1454) Provide User and Groups in Nextcloud Provider (@max-te)
```

Comment:
This will need moving as we have had a release since this was last updated

Reply:
will do